src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/templates/groups/Name.tsx

```diff
@@ -20,10 +21,10 @@ export default function Name({nodes, inputProps}: GroupProps) {
     return (
         <>
             {honorific &&
-                <NodeWrapper type={'text'} nodeType={'fields'} name={'honorific'}>
-                    <Text inputProps={inputProps['honorific']}
-                          fieldError={honorificError}
-                          {...honorific}
+                <NodeWrapper type={'select'} nodeType={'fields'} name={'honorific'}>
```

Comment:
If a prop is just a string, then it doesn't need the brackets: `type="select"`

Reply:
Yeah for some reason PHPstorm has not been auto formatting those.  Probably missing a code style setting somewhere.

Reply:
Found the setting in php storm html code style "Add for JSX attributes"

67058d08